packages/core/admin/admin/src/content-manager/components/DynamicTable/ConfirmDialogDeleteAll/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Dialog, DialogBody, DialogFooter } from '@strapi/parts/Dialog';
+import { Stack } from '@strapi/parts/Stack';
+import { Row } from '@strapi/parts/Row';
+import { Text } from '@strapi/parts/Text';
+import { Button } from '@strapi/parts/Button';
+import AlertWarningIcon from '@strapi/icons/AlertWarningIcon';
+import DeleteIcon from '@strapi/icons/DeleteIcon';
+import InjectionZoneList from '../../InjectionZoneList';
+import { getTrad } from '../../../utils';
+
+const ConfirmDialogDeleteAll = ({ isConfirmButtonLoading, isOpen, onToggleDialog, onConfirm }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Dialog
+      onClose={onToggleDialog}
+      title={formatMessage({
+        id: 'app.components.ConfirmDialog.title',
+        defaultMessage: 'Confirmation',
+      })}
+      labelledBy="confirmation"
+      describedBy="confirm-description"
+      isOpen={isOpen}
+    >
+      <DialogBody icon={<AlertWarningIcon />}>
+        <Stack size={2}>
+          <Row justifyContent="center">
+            <Text id="confirm-description">
+              {formatMessage({
+                id: getTrad('popUpWarning.bodyMessage.contentType.delete.all'),
+                defaultMessage: 'Are you sure you want to delete these entries?',
```

Comment:
would it be possible to pass the message through props to keep only one ConfirmDialogDelete component? or maybe I'm missing another difference between them 💭

Reply:
This one has an InjectionZone that allows the i18n plugin to display a message when trying to delete an entry. I think it's best no to update the current component but use the ones from parts instead.